spec/models/placements/mentor_membership_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe Placements::MentorMembership, type: :model do
+  context "with associations" do
+    it { is_expected.to belong_to(:mentor) }
+    it { is_expected.to belong_to(:school) }
```

Comment:
Ditto with this, re: module namespacing.

Reply:
c06a141d8659b694ad202fd79569d29eaeceaf2c